src/ui/default-prop-types.js

```diff
@@ -283,3 +282,9 @@ export const lessonMetadataShape = PropTypes.shape({
 export const lessonShape = PropTypes.shape({
   ...lessonMetadataProps
 });
+
+export const storageShape = PropTypes.shape({
```

Comment:
I think we have to be more specific with the naming to indicate that this is the config for the file picker. We will also have a storage shape on the user (the `storage` property) and a storage plan shape.

Reply:
Indeed, renamed to `filePickerStorageShape`, thanks!